### PR TITLE
In BREEZE.rst the command to create a user is outdated. #8086

### PR DESCRIPTION
**Pull request by ghost**
The command to create a user is outdated version.

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
Out of curiosity -> what was wrong with the original example? Looks the same, just different sequence of parameters.

File: BREEZE.rst [hunk 1]
```diff
@@ -482,7 +482,8 @@ running the webserver in order to log in. This can be done with the following co
 
 .. code-block:: bash
 
-    airflow users create --role Admin --username admin --password admin --email admin@example.com --firstname foo --lastname bar
+    airflow users create --email EMAIL --firstname FIRSTNAME --lastname LASTNAME --password PASSWORD --role Admin --username USERNAME
```
Comment (kaxil) [inline BREEZE.rst:485]: Except changing the order of args, what has changed?

Comment (zhongjiajie) [reply BREEZE.rst:485]: I check in and no other change, did old command don't work for you? @ghost

Comment (zhongjiajie) [reply BREEZE.rst:485]: Oh, a deleted user......

**Comment by zhongjiajie** [comment]
I prefer to close this cause the two command is work in my env, and the PR author seem deleted the account. Will open it if necessary

**Pull request closed without merging**